jquery.dynatable.js

```diff
@@ -315,6 +315,11 @@
 
       td += '"';
     }
+    
+    // allow cells to be editable
+    if (column.contentEditable) {
+      td += ' contenteditable="true"';
```

Comment:
Was aware that I didn't use the camelCase; I should have mentioned this earlier.
Though it isn't meant to be `contentEditable` http://www.w3.org/TR/2008/WD-html5-20080610/editing.html#contenteditable0 ?

(I can't edit this PR that since I removed the branch a long time ago...making a new one)